producer/src/main/kotlin/streams/procedures/StreamsProcedures.kt

```diff
@@ -22,31 +25,34 @@ class StreamsProcedures {
 
     @JvmField @Context var log: Log? = null
 
+    @Procedure(mode = Mode.READ, name = "streams.publish.sync")
+    @Description("streams.publish.sync(topic, payload, config) - Allows custom synchronous streaming from Neo4j to the configured stream environment")
+    fun sync(@Name("topic") topic: String?, @Name("payload") payload: Any?,
+             @Name(value = "config", defaultValue = "{}") config: Map<String, Any>?): Stream<StreamPublishResult> {
+        checkEnabled()
+        if (isTopicNullOrEmpty(topic)) {
+            return Stream.empty()
```

Comment:
I'm ok with this for now but I can imagine users will ask for this to throw an error instead.

Imagine this use case:

```
MATCH (k:KafkaConfig { id: 1 })
MATCH (p:Person)
WITH k.topic as topic, collect(p.name) as names
CALL streams.publish.sync(topic, names, {})
```

Now imagine that k.topic = ''.    This will *look like it did something* but will actually do nothing at all; in this case the user needs to inspect the output to see if it's non-empty to know if they succeeded.   But it isn't clear which error semantics are best here, so we continue as it is for now.